### DISCUSSION TITLE
Do roll-ups share source attribution?

### DISCUSSION THREAD

**Discussion by thefedoration** [plausible/analytics#1814] [Anything Else]
I currently have 2 plausible sites, one for my application and one for my marketing site. I like having them separate to be able to see the activity on both independently.

Unfortunately, this does provide a hole in my analytics. Users make important events (like purchasing) on my application, but the "Top Sources" mainly states that they came from my marketing site.

I would love to see the sources of visitors on my marketing site, against that same visitor making events on my application site. Is this possible?

I have just recently learned of [roll-up views](https://plausible.io/docs/plausible-script#is-there-a-roll-up-view), would this enable what I am looking for, or do I need to combine under 1 tracking snippet to see this information?

**Comment by metmarkosaric** [comment]
thanks @thefedoration! the roll up view should do what you need. it will keep the stats independent from each other on their individual dashboards but will also give you one new dashboard with combined stats where the referral source will be the original source rather than your marketing site. do try it out.

**Reply by thefedoration** [reply]
Well that is simply fantastic. Great feature, thanks Marko!

**Reply by metmarkosaric** [reply]
you're welcome! have fun!

**Reply by thefedoration** [reply]
@metmarkosaric unfortunately this does not seem to work as I'd expected, or if it does then the majority of the data is missing.

For example, for my subscription created event, the `top sources` section only has a value for about 1 in 7 goal conversions. This would be really valuable to have so that I know which data sources that bring users to my marketing site actually convert to paid users in the application site.

Is it a matter of time due to not using cookies? For example if the conversion happens over a week after someone lands on the marketing site, would that data be missing for a conversion on the application site?

**Reply by metmarkosaric** [reply]
have you clicked on the details button in "top sources"? that should then show you conversions from "direct/none" traffic too.

we only work with last click attribution because of our privacy-focus and because we don't use any long term identifiers. so if the conversion happens a week after someone first lands on your site, the referral source of the last visit when the conversion happened will be counted as the referral source of the conversion
